.github/workflows/test.yml

```
@@ -0,0 +1,26 @@
+# Inlcuding the tests with ubuntu to get actual test coverage. The OSX builds skip seccomp tests.
+name: test-coverage
+on: [push, pull_request]
+jobs:
+  validate:
+    name: Validate 
+    runs-on: ubuntu-latest
+    # Pull requests from the same repository won't trigger this checks as they were already triggered by the push
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    steps:
+      - name: Clone the code
+        uses: actions/checkout@v2
```

To be honest my personal preference would be to avoid the `name` field as much as we can. The actions name will be inherited by the `uses` as fallback, which already states that it will `checkout` the code.

```suggestion
      - uses: actions/checkout@v2
```